src/views/tables/PositionsTable.tsx

```diff
@@ -253,6 +259,14 @@ const getPositionsTableColumnDef = ({
           </TableCell>
         ),
       },
+      [PositionsTableColumnKey.Actions]: {
+        columnKey: 'actions',
+        label: stringGetter({ key: STRING_KEYS.ACTION }), // TODO: CT-639
+        isActionable: true,
+        allowsSorting: false,
+        hideOnBreakpoint: MediaQueryKeys.isTablet,
+        renderCell: ({}) => <PositionsActionsCell isDisabled={isAccountViewOnly} />,
```

Comment:
technically `isDisabled={isAccountViewOnly}` shouldn't ever be necessary since our callers don't pass the `Actions` column key when `isAccountViewOnly`; but I think it's probably better to have an extra check here (and matches what happens with `OrdersTable` currently